src/fortranscanner.l

```diff
@@ -2839,7 +2839,7 @@ static void scanner_abort()
 #if !defined(YY_FLEX_SUBMINOR_VERSION) 
 //----------------------------------------------------------------------------
 extern "C" { // some bogus code to keep the compiler happy
-  void fortranscannernerYYdummy() { yy_flex_realloc(0,0); } 
+  void fortranscannerYYdummy() { yy_flex_realloc(0,0); } 
```

Comment:
I know this is a dummy thing, but I don't see any particular reason for the `ner` suffix.

Reply:
Neither do I.